test/integration/world_tagging_flow.rb

```diff
@@ -0,0 +1,66 @@
+require 'test_helper'
+require 'capybara/rails'
+
+class WorldTaggingFlow < ActionDispatch::IntegrationTest
+  extend Minitest::Spec::DSL
+  include Capybara::DSL
+  include TaxonomyHelper
+  include Rails.application.routes.url_helpers
+
+  setup do
+    login_as_admin
+  end
+
+  context 'given I want to tag to the WorldWide taxonomy' do
+    let(:world_organisation) do
+      create(:organisation, content_id: "9adfc4ed-9f6c-4976-a6d8-18d34356367c")
+    end
+    let(:world_edition) do
+      create(:publication, :guidance, organisations: [world_organisation])
+    end
+
+    before do
+      setup_world_tagging_page
+    end
+
+    it "asserts WorldWide tagging path" do
+      visit_world_tags_path
+      assert_equal(200, page.status_code)
+    end
+
+    it "asserts a selected checkbox' " do
+      visit_world_tags_path
+      expand_and_check_checkbox
+      assert page.has_checked_field?('World grand child taxon')
```

Comment:
I'm a bit surprised this is still working, as the taxon has now been changed to "grandchild"...

Reply:
Urmm, weird! I've ran it locally and it failed, i've corrected the test accordingly.